finetuner/__init__.py

```diff
@@ -267,16 +267,16 @@ def get_run(run_name: str, experiment_name: Optional[str] = None) -> Run:
     return ft.get_run(run_name=run_name, experiment_name=experiment_name)
 
 
-def list_runs(experiment_name: Optional[str] = None) -> List[Run]:
+def list_runs(experiment_name: Optional[str] = None, size: int = 50) -> List[Run]:
     """List every run.
 
     If an experiment name is not specified, we'll list every run across all
     experiments.
 
     :param experiment_name: Optional name of the experiment.
-    :return: A list of `Run` objects.
+    :param size: Number of runs to retrieve.
```

Comment:
so I get the first 50 sending a request with size=50. How do I get the next 50?

Reply:
set a larger size, each page will give your more items, e.g. 100

Reply:
what I'm basically saying is that we need to expose `page` as well

Reply:
I think a page parameter is missing right?

Reply:
page added